database/sql_handler.go

```diff
@@ -31,3 +31,11 @@ func GetUser(ctx context.Context, token string) (string, error) {
 }
 
 // tokenとnameを受け取ってtokenに該当するuserのnameを更新する
+func UpdateUser(ctx context.Context, token string, name string) error {
+	_, err := DB.QueryContext(ctx, "UPDATE users SET name = ? WHERE token = ? ", name, token)
+	if err != nil {
+		log.Printf("failed to update user: %v", err)
+		return err
+	}
```

Comment:
省略して書けるよ！
こっちの方がerrの変数のスコープがifブロック内だけになる

```suggestion
	if _, err := DB.QueryContext(ctx, "UPDATE users SET name = ? WHERE token = ? ", name, token); err != nil {
		log.Printf("failed to update user: %v", err)
		return err
	}
```

Reply:
>こっちの方がerrの変数のスコープがifブロック内だけになる

なるほど！！知見です

Reply:
[1ccc2ea](https://github.com/fumist23/game-api/pull/4/commits/1ccc2eae972b11cbf1d255701e1f68033ccfa300)で修正しましたmm